client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -267,6 +298,34 @@ export function handlePage( action, data ) {
 			feed_ID: post.feed_ID,
 			xPostMetadata: XPostHelper.getXPostMetadata( post ),
 		} ) );
+	} else if ( cardPosts && cardRecommendedSites ) {
+		streamItems = cardPosts.map( ( post ) => ( {
+			...keyForPost( post ),
+			date: post[ dateProperty ],
+			...( post.comments && { comments: map( post.comments, 'ID' ).reverse() } ), // include comments for conversations
+			url: post.URL,
+			site_icon: post.site_icon?.ico,
+			site_description: post.description,
+			site_name: post.site_name,
+			feed_URL: post.feed_URL,
+			feed_ID: post.feed_ID,
+			xPostMetadata: XPostHelper.getXPostMetadata( post ),
+		} ) );
+
+		streamSites = cardRecommendedSites.map( ( site ) => {
+			return {
+				url: site.URL,
+				site_icon: site.icon?.ico,
+				site_description: site.description,
+				site_name: site.name,
+				feed_URL: site.feed_URL,
+				feed_ID: site.feed_ID, // Not sure if this is needed yet
```

Comment:
it sounds like this is what we needed for recieveRecommendedSites? and do we also need the `feedId` version below as well?